### DISCUSSION TITLE
Please add buymeacoffee.com as a plattform option in funding.yaml

### DISCUSSION THREAD

**Discussion by jhlothamer** [community/community#112012] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I would like to see buymeacoffee.com added as a platform option in the funding.yaml.  It's a popular alternative to Ko-fi, and it would be good to see a nicely formated link along with a logo icon just like we see for Kofi instead of the custom URL treatment.  Also, links under the "Sponsor this project" section will open a new tab, which is something we cannot do in a readme for a github project.

**Comment by octokatherine** [comment]
Hi @jhlothamer! Thanks for the suggestion. We've added support for buymeacoffee.com as a funding platform option.

**Reply by jhlothamer** [reply]
Thanks so much!!
